### PR TITLE
Add reminder for Linting in PR template

### PR DESCRIPTION
**Pull request by xinyis991105**
## Description
Add reminder for linting in PR template's checklist section.

## Motivation and Context
This is in an effort to enforce linting standard on future PR's.

## How Has This Been Tested?
N/A

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation, which is located at [Autolab Docs](https://github.com/autolab/docs)
- [ ] I have updated the documentation accordingly, included in this PR

## Other issues / help required

If unsure, feel free to submit first and we'll help you along.

### CODE REVIEW

**Review by fanpu — APPROVED** [review]
Looks good, approving this first. Am I right to say that we'll only merge this once Victor is completely done with his rubocop linting fixes? Thanks!

**Comment by xinyis991105** [comment]
Exactly! I won't merge this in until Victor has merged in all of the linting fixes. Thanks for reviewing it!

**Comment by xinyis991105** [comment]
Linting is done and reminder has been added in another commit.

**Pull request closed without merging**
